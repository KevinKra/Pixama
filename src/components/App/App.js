import React from "react";
import { Route } from "react-router-dom";
import NavBar from "../NavBar/NavBar";
import MainPage from "../MainPage/MainPage";
import Carousel from "../Carousel/Carousel";

export default function App() {
  return (
    <main>
      <Route path="/" component={NavBar} />
      <Route exact path="/" component={MainPage} />
    </main>
  );
}
