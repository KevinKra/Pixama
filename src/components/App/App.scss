@import "../../_styles//variables.scss";

.App {
  background-color: $primary-color;
}
