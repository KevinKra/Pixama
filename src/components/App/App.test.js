import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
