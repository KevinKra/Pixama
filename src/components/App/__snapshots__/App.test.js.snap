// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match a snapshot 1`] = `
<main
  className="App"
>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(NavBar)",
        "type": [Function],
      }
    }
    path="/"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(MainPage)",
        "proptyes": Object {
          "addPopularMovies": [Function],
          "addRomanceMovies": [Function],
          "favorites": [Function],
          "popularMovies": [Function],
          "romanceMovies": [Function],
          "updateFavorites": [Function],
        },
        "type": [Function],
      }
    }
    exact={true}
    path="/"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(MoviePage)",
        "type": [Function],
      }
    }
    exact={true}
    path="/moviepage"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(LoginCard)",
        "type": [Function],
      }
    }
    exact={true}
    path="/login"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(RegisterCard)",
        "type": [Function],
      }
    }
    exact={true}
    path="/register"
  />
</main>
`;
